source/mir/random/engine/test.d

```diff
@@ -0,0 +1,232 @@
+module mir.random.engine.test;
+
+import std.typecons;
+import std.stdio;
+
+version(linux) version(X86_64)
+{
+    shared static this()
+    {
+        hasGetRandom = initHasGetRandom;
+    }
+
+    private static bool hasGetRandom;
+
+    auto initHasGetRandom()
+    {
+        import core.sys.posix.sys.utsname;
+        utsname uts;
+        uname(&uts);
+
+        import std.algorithm.iteration : splitter;
+        import std.conv : to;
```

Comment:
Please avoid dependencies from std.algorithm and std.conv
C functions can be used instead